docs/install/installation.rst

```diff
@@ -0,0 +1,118 @@
+.. meta::
+   :description: HSA runtime implementation
+   :keywords: ROCR, ROCm, library, tool, runtime
+
+.. _installation:
+
+====================
+Installation
+====================
+
+This document provides information required to build and install ROCR using prebuilt binaries or from source.
+
+Build and install using prebuilt binaries
+-------------------------------------------
+
+Here is how you can install ROCR using prebuilt binaries.
+
+Prerequisites
+*******************
+
+- A system supporting ROCm. See the `supported operating systems <https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html#supported-operating-systems>`_.
+
+- Install ROCm. See `how to install ROCm <https://rocm.docs.amd.com/projects/install-on-linux/en/latest/>`_.
+
+- Install ``libdrm`` package.
```

Comment:
@dayatsin-amd Please provide command to install libdrm if required.

Reply:
sudo apt install libdrm-dev